libraries/chain/protocol_feature_manager.cpp

```diff
@@ -258,6 +258,17 @@ Adds new cryptographic host functions
 Builtin protocol feature: GET_BLOCK_NUM
 
 Enables new `get_block_num` intrinsic which returns the current block number.
+*/
+            {}
+         } )
+         (  builtin_protocol_feature_t::bls_primitives, builtin_protocol_feature_spec{
+            "BLS_PRIMITIVES",
+            fc::variant("2c302889ce2db124878f7c6092cf27519d450559ed2fdfbad14f532d90fc5139").as<digest_type>(),
```

Comment:
This SHA256 hash value of the comment message below is probably incorrect. I haven't been able to reproduce any of the other hashes, so I assume the one I generated is probably wrong.

Reply:
The SHA256 hash includes the trailing newline.
For example, GET_BLOCK_NUM, hash the string between the /**/, quoted below, including the trailing newline but not the newline before "Builtin".
```
"Builtin protocol feature: GET_BLOCK_NUM

Enables new `get_block_num` intrinsic which returns the current block number.
"
```

Reply:
So yours would be `01969c44de35999b924095ae7f50081a7f274409fdbccb9fc54fa7836c76089c`

Reply:
Hello Kevin. Thank you very much for clarifying that. I tried a few different things but not that. I am now able to reproduce all the hashes.

Corrected the hash value.